stores/searchStore.js

```diff
@@ -19,11 +19,12 @@ class SearchStore {
     @action searchTMDBforMovieByName = query => {
         const movieSearcher = movieApi({ tmdbApiKey: TMDB_API_KEY, omdbApiKey: OMDB_API_KEY });
 
-        const movieQuery = `https://api.themoviedb.org/3/discover/movie?api_key=${TMDB_API_KEY}&language=en-US&sort_by=vote_average.desc&include_adult=false&include_video=false&page=1&vote_count.gte=1000&vote_average.gte=8`;
+        // const movieQuery = `https://api.themoviedb.org/3/discover/movie?api_key=${TMDB_API_KEY}&language=en-US&sort_by=vote_average.desc&include_adult=false&include_video=false&page=1&vote_count.gte=1000&vote_average.gte=8`;
```

Comment:
This query picks the most popular movies. It wasn't using the actual query string, but leaving it here if I want to use this particular query string in the future.